specs/backend-api.graphql

```diff
@@ -73,12 +73,14 @@ type ApplicationVerificationView {
 }
 
 type ApplicationView {
-  cardCreated: Boolean
+  cardCreated: Boolean @deprecated(reason : "Use 'status' instead")
```

Comment:
should be keep this field ?

Reply:
The field will be removed in the follow-up PR #2276